01-glasswork/2023-front-end-html.html

```diff
@@ -697,6 +697,25 @@ <h1>Resources to Look It Up</h1>
             <li class="fragment">Peers: ask co-workers or KCWiT on Slack!</li>
           </ul>
         </section>
+        <section>
+          <h1>Classic way to Debug</h1>
+          <br><br>
+          <ul style="list-style:disc; margin-left: 80px;">
+            <li class="fragment">Comment out code</li>
+            <br>
+            <li class="fragment">Uncomment code</li>
+            <br>
+            <li class="fragment">Repeat until you find the issue</li>
+          </ul>
+        </section>
+        <section>
+          <h1>Classic Way to Debug</h1>
+          <br><br>
+          <ul style="list-style:disc; margin-left: 80px;">
+            <li class="fragment">Rubber ducky</li>
+          </ul>
+          <!-- TODO: Add rubber ducky pic -->
```

Comment:
do you have a ticket number that we could add to this inline comment?

Reply:
For you, I'll make a ticket number. 🚀